tests/unit/amo/components/TestAddon.js

```diff
@@ -768,6 +810,268 @@ describe('Addon', () => {
       );
     });
   });
+
+  describe('more add-ons by authors', () => {
+    const dispatchLoadAddonsAndOtherAddonsByAuthors = ({
```

Comment:
I'd say you can just call this `dispatchAddonData` since it's only used within the context of the 'more add-ons by authors' test.